api/src/__init__.py

```diff
@@ -0,0 +1,121 @@
+import requests
+from typing import List
+
+# ---------- GitHub API Classes ----------
+
+class GitHubIssue:
+    def __init__(self, issue_id, title, body, url, state):
+        self.id = issue_id
+        self.title = title
+        self.body = body
+        self.url = url
+        self.state = state
+
+class GitHubAPI:
+    def __init__(self, token: str):
+        self.token = token
+        self.headers = {
+            "Authorization": f"token {self.token}",
+            "Accept": "application/vnd.github+json"
+        }
+
+    def fetch_issues(self, owner: str, repo: str) -> List[GitHubIssue]:
+        url = f"https://api.github.com/repos/{owner}/{repo}/issues"
+        response = requests.get(url, headers=self.headers)
+        response.raise_for_status()
+        return [
+            GitHubIssue(
+                issue_id=item["number"],
+                title=item["title"],
+                body=item.get("body", ""),
+                url=item["html_url"],
+                state=item["state"]
+            )
+            for item in response.json()
+        ]
+
+    def close_issue(self, owner: str, repo: str, issue_id: int) -> bool:
+        url = f"https://api.github.com/repos/{owner}/{repo}/issues/{issue_id}"
+        data = {"state": "closed"}
+        response = requests.patch(url, headers=self.headers, json=data)
+        return response.status_code == 200
+
+    def comment_on_issue(self, owner: str, repo: str, issue_id: int, comment: str) -> bool:
+        url = f"https://api.github.com/repos/{owner}/{repo}/issues/{issue_id}/comments"
+        response = requests.post(url, headers=self.headers, json={"body": comment})
+        return response.status_code == 201
+
+
+# ---------- Trello API Classes ----------
+
+class TrelloCard:
```

Comment:
Creating new types for tasks is good, but they shouldn't be named based on the backend. It should be named something like `Task` or `Issue` or something like that.